ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PeerSimulator/Tests/Timeouts.hs

```diff
@@ -61,9 +62,11 @@ prop_timeouts = do
     dullSchedule :: DiffTime -> TestFrag -> Peers PeerSchedule
     dullSchedule _ (AF.Empty _) = error "requires a non-empty block tree"
     dullSchedule timeout (_ AF.:> tipBlock) =
-      let tickDuration = 1 -- 1s
+      let tickDuration = secondsToDiffTime 1 -- 1s
           maximumNumberOfTicks = round $ timeout / tickDuration
        in peersOnlyHonest $
             (Time 0, ScheduleTipPoint tipBlock)
               : (Time 0, ScheduleHeaderPoint tipBlock)
-              : zip (map (Time . (* tickDuration)) [0..]) (replicate maximumNumberOfTicks (ScheduleBlockPoint tipBlock))
+              : zip
+                  (map (Time . (* tickDuration) . secondsToDiffTime) [0..])
+                  (replicate (1 + maximumNumberOfTicks) (ScheduleBlockPoint tipBlock))
```

Comment:
Instead of many ticks, maybe the schedule could have just one `ScheduleBlockPoint` at time `maximumNumberOfTicks*tickDuration`.

Reply:
Yes, that would work too, and maybe make the code simpler.